packages/apputils/src/toolbar.tsx

```diff
@@ -440,11 +440,12 @@ export function ToolbarButtonComponent(props: ToolbarButtonComponent.IProps) {
   // We avoid a click event by calling preventDefault in mousedown, and
   // we bind the button action to `mousedown`.
   const handleMouseDown = (event: React.MouseEvent) => {
-    // Fire action only when left button is pressed.
-    if (event.button === 0) {
-      event.preventDefault();
-      props.onClick();
-    }
+    event.preventDefault();
```

Comment:
We should probably still check for the `button === 0` both here and in mouse up.